src/main/java/zerobase/boardproject/dto/CreatePost.java

```diff
@@ -0,0 +1,49 @@
+package zerobase.boardproject.dto;
+
+import com.sun.istack.NotNull;
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+public class CreatePost {
+
+  @Getter
+  @Setter
+  @AllArgsConstructor
+  public static class Request {
+
+    @NotNull
+    private BigInteger user_id;
```

Comment:
```suggestion
    private BigInteger userId;
```

camelCase 로 지정할 수 있도록 해주세요. 아래 response 영역도 동일합니다.